Ginger/GingerCoreNET/Database/DatabaseOperations.cs

```diff
@@ -59,7 +59,28 @@ ValueExpression VE
         {
             get
             {
-                 return mVE = new ValueExpression(Database.ProjEnvironment ?? new ProjEnvironment(), Database.BusinessFlow ?? new BusinessFlow(), Database.DSList);
+                Database.ProjEnvironment ??= new ProjEnvironment();
```

Comment:
changing properties in getter is not recommended, required properties should be set in setter and should be handled when object of database operation is created